src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute in the link tag is pointing to a SCSS file. It should be pointing to a CSS file instead, as browsers do not understand SCSS directly. Make sure to compile your SCSS to CSS and link the CSS file here.